sds_data_manager/lambda_images/instruments/dependents.json

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
Maybe rename this file to "upstream_dependencies.json"?

Reply:
Also, a comment on the structure of this file is probably a good thing to add, because there was a fair amount of confusion about it